provisioner.go

```diff
@@ -342,20 +352,34 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 				{
 					Name:    "local-path-" + string(action),
 					Image:   p.helperImage,
-					Command: append(cmdsForPath, filepath.Join("/data/", volumeDir)),
+					Command: cmdsForPath,
+					Args:    []string{filepath.Join(parentDir, volumeDir), strconv.FormatInt(sizeInBytes, 10)},
 					VolumeMounts: []v1.VolumeMount{
 						{
 							Name:      "data",
 							ReadOnly:  false,
-							MountPath: "/data/",
+							MountPath: parentDir,
 						},
 						{
 							Name:      "script",
 							ReadOnly:  false,
 							MountPath: "/script",
 						},
+						{
+							Name:      "xfs-quota-projects",
```

Comment:
Hmm, those are too specific for XFS. Is there any way we can make it more generic?